swift/ql/lib/codeql/swift/controlflow/internal/ControlFlowGraphImpl.qll

```diff
@@ -911,9 +908,9 @@ module Exprs {
         child.asAstNode() = ast.getSource().getFullyConverted()
       }
 
-      predicate hasWillSetObserver() { isPropertyObserverElement(any(WillSetObserver obs), ast) }
+      predicate hasWillSetObserver() { isPropertyObserverElement(_, any(WillSetObserver obs), ast) }
 
-      predicate hasDidSetObserver() { isPropertyObserverElement(any(WillSetObserver obs), ast) }
+      predicate hasDidSetObserver() { isPropertyObserverElement(_, any(WillSetObserver obs), ast) }
```

Comment:
Now that I look at this again, shouldn't this be `any(DidSetObserver obs)`?

Reply:
Oh, oops. You're right. It definitely should

Reply:
Fixed in c7cc8d2592ed292ad09692bc187d513ab7ced158.